docs/UserManuals/ConfigUI/Jenkins.md

```diff
@@ -59,6 +59,14 @@ If you're using Jenkins builds to conduct `deployments`, please select "Detect D
 - Deployment: A Jenkins build with a name that matches the given regEx will be considered as a deployment.
```

Comment:
I recommend revising to below:

If you'd like to define `deployments` with Jenkins, please select "Detect Deployment from Jenkins Builds", and provide the following regexes

- Deployment: Jenkins stages whose names match the regex will be registered as deployments (if a Jenkins job has no stage, its job name will be used to match the regex)
- Production: Jenkins stages whose names match the regex will be assigned environment 'PRODUCTION' (if a Jenkins job has no stage, its job name will be used to match the regex)

Reply:
fixed

Reply:
Also, we need to say` if a Jenkins build has no stage`